### ISSUE TITLE
Hylailos-Stil in Verbindung mit Pailos

### ISSUE THREAD

**Issue by mwberlindeu** [elyukai/optolith-client#743]
Der Hylailos-Stil senkt die PA-Abzüge von AT/PA-Modifikatoren bei Stangenwaffen.
https://ulisses-regelwiki.de/index.php/KS_Hylailos.html

Ich habe in Verbindung mit einem Pailos geprüft, ob die Absenkung vorhanden ist, was ich aber in Version 1.4.0 nicht nachvollziehen konnte.

Ich vermute, dass die Auswirkungen von Kampfstilen auf Waffenwerte ggf. insgesamt noch nicht implementiert sind.

Auswirkungen von Kampfstilen sollten errechnet werden.

Alternativ können die Waffenwerte selbst natürlich angepasst werden. Der Nachteil hierbei ist, dass die Information verloren gehen kann, ob sich ein bestimmter Effekt auf einen Kampfstil oder ggf. eine besondere (magische) Waffe bezieht.

**Comment by elyukai** [comment]
Genau, Auswirkungen werden generell nicht angerechnet. Das wollte ich bei Item Sets (#59) mitliefern, da diverse Anrechnungen gar nicht verlässlich möglich sind, ohne genau zu wissen, welche und wie Rüstung(en) und Waffe(n) getragen werden. Aktuell wird nur Kampftechnik-AT/PA mit dem AT/PA-Mod der Waffe verrechnet.
